lib/dark_matter/decimals/conversion.ex

```diff
@@ -45,11 +45,11 @@ defmodule DarkMatter.Decimals.Conversion do
   end
 
   def cast_decimal(%Decimal{} = decimal, :reduced) do
-    decimal |> Decimal.reduce()
+    decimal |> Decimal.normalize()
   end
 
   def cast_decimal(%Decimal{} = decimal, :normal) do
-    {:ok, normal_decimal} = decimal |> Decimal.to_string(:normal) |> Decimal.parse()
+    {normal_decimal, _remainder} = decimal |> Decimal.to_string(:normal) |> Decimal.parse()
```

Comment:
This was the only significant API change I found

Reply:
🔥